Contents/Code/constants.py

```diff
@@ -98,3 +99,27 @@
         base_url, issue_label, issue_template, title_prefix['movie_collections'], '{}', url_name,
         url_prefix['movie_collections'], '{}'),
 )
+
+media_type_dict = dict(
```

Comment:
was this meant to be part of the other PR?

Reply:
ah you're right, I committed the whole constants.py file by mistake